app/styles/pages/group-index.scss

```diff
@@ -0,0 +1,63 @@
+@mixin panel() {
+  background-color: $white;
+  border-color: $gray-light;
+  border-style: solid;
+  border-width: 1px;
+  padding: $padding-m;
+  margin-bottom: $padding-m;
+}
+
+.group-index {
+  background-color: $gray-extralight;
+
+  .group-banner {
+    background-position: center;
+    background-size: cover;
+    color: $white;
+    margin-bottom: $padding-m;
+    padding: 250px $padding-l $padding-l $padding-l;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `250px $padding-l $padding-l` instead of `250px $padding-l $padding-l $padding-l`